From de80487ed2ce3d935c2ad972462f4087cf938407 Mon Sep 17 00:00:00 2001
From: Julien Cristau <jcristau@debian.org>
Date: Thu, 26 Aug 2010 19:48:57 +0200
Subject: [PATCH] Comment out std::ostream &operator<<(std::ostream &out, const pkgCache::DepIterator &dep)

This clashes with the function provided by libapt-pkg in 0.8.0.
---
 src/generic/apt/aptitude_resolver.cc |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/generic/apt/aptitude_resolver.cc b/src/generic/apt/aptitude_resolver.cc
index 6331fb3..87d2588 100644
--- a/src/generic/apt/aptitude_resolver.cc
+++ b/src/generic/apt/aptitude_resolver.cc
@@ -85,6 +85,7 @@ namespace
   }
 }
 
+#if 0
 // Logging operators.
 std::ostream &operator<<(std::ostream &out, const pkgCache::DepIterator &dep)
 {
@@ -122,6 +123,7 @@ std::ostream &operator<<(std::ostream &out, const pkgCache::DepIterator &dep)
 
   return out;
 }
+#endif
 
 // Should version selections be logged the way they're written?
 // That's a little awkward since the syntax is hairy and some of them
-- 
1.7.1

